package.json

```diff
@@ -232,5 +236,13 @@
   "bugs": {
     "url": "https://github.com/Trustroots/trustroots/issues"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "jest": {
```

Comment:
As the package.json is getting longer, this could also go to a [separate config](https://jestjs.io/docs/en/configuration) (jest.config.js).
Just an info. Optional. :)

Reply:
I also think separate config file would be helpful for better discoverability.

Reply:
Done!